### ISSUE TITLE
Incompability with Optifine and latest forge 1.11.2 when opening inventory

### ISSUE THREAD

**Issue by Andyghandi** [mezz/JustEnoughItems#714]
Was updating this mod and optifine along with some others, but when only playing with this mod and optifine, the game crashes when trying to open inventory. If running these 2 mods seperately, it all just works fine! But they won't cooperate, please have a look at the crash report -->

Description: Unexpected error

java.lang.IllegalStateException: Already building!
	at net.minecraft.client.renderer.VertexBuffer.func_181668_a(VertexBuffer.java:291)
	at net.minecraft.client.shader.Framebuffer.func_178038_a(Framebuffer.java:264)
	at net.minecraft.client.shader.Framebuffer.func_147615_c(Framebuffer.java:229)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1104)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:372)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at net.minecraft.client.renderer.VertexBuffer.func_181668_a(VertexBuffer.java:291)
	at net.minecraft.client.shader.Framebuffer.func_178038_a(Framebuffer.java:264)

**Comment by cleavenworth** [comment]
I'm getting the exact same issue and crash log. Here's some potentially relevant client log that I didn't see in my own crash report or this one.
```[23:23:54] [Client thread/ERROR] [FML/]: Exception caught during firing event net.minecraftforge.client.event.GuiScreenEvent$BackgroundDrawnEvent@33240029:
java.lang.NoSuchFieldError: field_78286_d
	at net.minecraftforge.client.model.SimpleModelFontRenderer.func_78266_a(SimpleModelFontRenderer.java:68) ~[SimpleModelFontRenderer.class:?]
	at net.minecraft.client.gui.FontRenderer.func_181559_a(FontRenderer.java:293) ~[bfg.class:?]
	at net.minecraft.client.gui.FontRenderer.func_78255_a(FontRenderer.java:562) ~[bfg.class:?]
	at net.minecraft.client.gui.FontRenderer.func_180455_b(FontRenderer.java:675) ~[bfg.class:?]
	at net.minecraft.client.gui.FontRenderer.func_175065_a(FontRenderer.java:420) ~[bfg.class:?]
	at net.minecraft.client.gui.FontRenderer.func_78276_b(FontRenderer.java:399) ~[bfg.class:?]
	at net.minecraftforge.client.model.FancyMissingModel$BakedModel.func_188616_a(FancyMissingModel.java:146) ~[FancyMissingModel$BakedModel.class:?]
	at net.minecraft.client.renderer.RenderItem.func_175045_a(RenderItem.java:168) ~[bvh.class:?]
	at net.minecraft.client.renderer.RenderItem.func_175036_a(RenderItem.java:140) ~[bvh.class:?]
	at mezz.jei.gui.ingredients.GuiIngredientFast.renderItemAndEffectIntoGUI(GuiIngredientFast.java:120) ~[GuiIngredientFast.class:?]
	at mezz.jei.gui.ingredients.GuiIngredientFast.renderItemAndEffectIntoGUI(GuiIngredientFast.java:93) ~[GuiIngredientFast.class:?]
	at mezz.jei.gui.ingredients.GuiIngredientFastList.render(GuiIngredientFastList.java:165) ~[GuiIngredientFastList.class:?]
	at mezz.jei.gui.ItemListOverlayInternal.drawScreen(ItemListOverlayInternal.java:304) ~[ItemListOverlayInternal.class:?]
	at mezz.jei.GuiEventHandler.onDrawBackgroundEventPost(GuiEventHandler.java:89) ~[GuiEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_291_GuiEventHandler_onDrawBackgroundEventPost_BackgroundDrawnEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:185) [EventBus.class:?]
	at net.minecraft.client.gui.GuiScreen.func_146276_q_(GuiScreen.java:584) [bho.class:?]
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:73) [big.class:?]
	at com.raoulvdberge.refinedstorage.gui.GuiBase.func_73863_a(GuiBase.java:103) [GuiBase.class:?]
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:382) [ForgeHooksClient.class:?]
	at sun.reflect.GeneratedMethodAccessor25.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_25]
	at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_25]
	at Reflector.callVoid(Reflector.java:530) [Reflector.class:?]
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1414) [bqe.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1077) [bes.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:372) [bes.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_25]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_25]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_25]
	at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_25]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
[23:23:54] [Client thread/ERROR] [FML/]: Index: 1 Listeners:
[23:23:54] [Client thread/ERROR] [FML/]: 0: NORMAL
[23:23:54] [Client thread/ERROR] [FML/]: 1: ASM: mezz.jei.GuiEventHandler@60734e6f onDrawBackgroundEventPost(Lnet/minecraftforge/client/event/GuiScreenEvent$BackgroundDrawnEvent;)V

**Comment by TehNut** [comment]
https://github.com/sp614x/optifine/issues/438

**Comment by mezz** [comment]
Thanks for the report(s)!
Looks like this has been fixed in Optifine preview B7 for 1.11.2, it is not a JEI issue.

**Comment by archonsd** [comment]
Any chance this gets (or got?) backported to 1.10.2 Opti?  There's a D7 available in March that post-dates the 1.11.2 change referenced above, but I don't see it referenced in the D7 changelog and I'm getting this exact crash in 1.10.2 with Opti U D7, forge-1.10.2-12.18.3.2281 and JEI jei_1.10.2-3.14.7.416.

https://pastebin.com/jNLG7q7f

I pulled the middle-mod in the traceback, ReCubed, and it still crashes on search in JEI with the same error.  (possibly the missing texture remains in the inventory even with the mod removed since it's SMP and the world wasn't blown away)

**Comment by mezz** [comment]
I don't know, please ask in the optifine issue: sp614x/optifine#438
